glean-core/android/src/test/java/mozilla/telemetry/glean/pings/DeletionPingTest.kt

```diff
@@ -133,6 +134,7 @@ class DeletionPingTest {
             serverEndpoint = "http://" + server.hostName + ":" + server.port
         ), clearStores = true, uploadEnabled = false)
         triggerWorkManager(context)
+        waitForEnqueuedWorker(context, PingUploadWorker.PING_WORKER_TAG)
```

Comment:
Oh was this causing test failures?

Reply:
Yeap, this is a hacky fix for now. I figure this is a potential problem for all tests that use the `triggerWorkManager` function and I opened [a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1657589) to investigate later. Could really even be a problem in other bindings.